os/board/rtl8721csm/src/component/common/bluetooth/realtek/sdk/board/amebad/src/bt_board.h

```diff
@@ -0,0 +1,117 @@
+/**
+ * Copyright (c) 2015, Realsil Semiconductor Corporation. All rights reserved.
+ *
+ */
+#ifndef _BOARD_H_
+#define _BOARD_H_
+
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+#include <stdio.h>
+#include "ameba_soc.h"
+
+#define platform_debug DBG_8195A
+
+#define TRACE_SWITCH_CLOSE
+//trace_uart
+//========================================
+/* CONFIG */
+//#define TRACE_UART_DEFAULT 
+#define TRACE_UART_TX_IRQ
+//#define TRACE_UART_TX_WHILE
+//#define TRACE_UART_DMA
```

Comment:
same as above.
please correct all the files as mentioned.

Reply:
All the files mentioned above has been corrected, also the comment format has been modified.